src/azure-cli/azure/cli/command_modules/acr/_archive_utils.py

```diff
@@ -66,13 +66,19 @@ def _pack_source_code(source_location, tar_file_path, docker_file_path, docker_f
     original_docker_file_name = os.path.basename(docker_file_path.replace("\\", os.sep))
     ignore_list, ignore_list_size = _load_dockerignore_file(source_location, original_docker_file_name)
     common_vcs_ignore_list = {'.git', '.gitignore', '.bzr', 'bzrignore', '.hg', '.hgignore', '.svn'}
+    build_ignore_dirs = {'.venv'}
 
     def _ignore_check(tarinfo, parent_ignored, parent_matching_rule_index):
         # ignore common vcs dir or file
         if tarinfo.name in common_vcs_ignore_list:
```

Comment:
I think it's a latent bug. `tarinfo.name` is the full file path name in the tar. We need to extract the dir or file name like you does `os.path.basename(tarinfo.name)`.

Reply:
updated tarinfo.name